extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantConfigContext.java

```diff
@@ -47,6 +47,9 @@ private static SecretKey createPkceSecretKey(OidcTenantConfig config) {
         if (config.authentication.pkceRequired.orElse(false)) {
             String pkceSecret = config.authentication.pkceSecret
                     .orElse(OidcCommonUtils.clientSecret(config.credentials));
+            if (pkceSecret == null) {
+                throw new RuntimeException("Secret key for encrypting PKCE code verifier is missing");
+            }
```

Comment:
I think the method below would have the same issue, no?

Reply:
Ah good catch, let me add another commit fixing that